sdk/containerregistry/container-registry/swagger/containerregistry.json

```diff
@@ -1561,15 +1497,13 @@
             "configMediaType": "configMediaType",
             "lastUpdateTime": "lastUpdateTime",
             "architecture": "architecture",
-            "tags": [
-              "tags",
-              "tags"
-            ]
+            "tags": ["tags", "tags"]
           }
         ]
       }
     },
     "ManifestAttributes": {
+      "x-ms-client-name": "RegistryArtifactProperties",
```

Comment:
Would it possible to add a property that is a collection of `RegistryArtifactProperties` here?  This is currently defined as a `ManifestAttributes_manifest_references` array in `ManifestAttributes_manifest` in the swagger file, and returned from `GET /acr/v1/{name}/_manifests/{digest}` if the manifest is a "manifest list"

Let me know if this is something we'll want to handle in the customization layer instead.

Reply:
Good point!  I am looking into adding the array. It would be hard to achieve the desire type of `RegistryArtifactProperties` using transformation so we still need additional handling.